tests/dj_angles/mappers/thirdparty/test_map_bird_component.py

```diff
@@ -0,0 +1,54 @@
+from dj_angles.mappers.thirdparty import map_bird_component
+from tests.dj_angles.tags import create_tag
+
+
+def override_setting(settings, key, value):
```

Comment:
@adamghill not sure why I would have to use those 2 helps, this is probably something pytest should be doing for me but if I don't do that, a bunch of other tests start failing (because of the overridden setting)